Source/WebCore/dom/Node.h

```diff
@@ -797,12 +790,14 @@ class Node : public EventTarget, public CanMakeCheckedPtr<Node> {
     mutable uint32_t m_refCountAndParentBit { s_refCountIncrement };
     const uint16_t m_typeBitFields;
     mutable OptionSet<StateFlag> m_stateFlags;
+    mutable OptionSet<ElementStateFlag> m_elementStateFlags;
+    mutable StyleBitfields m_styleBitfields;
 
     CheckedPtr<ContainerNode> m_parentNode;
     TreeScope* m_treeScope { nullptr };
-    CompactPointerTuple<Node*, uint16_t> m_previous;
+    Node* m_previousSibling;
     CheckedPtr<Node> m_next;
-    CompactPointerTuple<RenderObject*, uint16_t> m_rendererWithStyleFlags;
+    RenderObject* m_renderer;
```

Comment:
Ditto.

Reply:
This got a `{ nullptr }` in the latest push as well.